### ISSUE TITLE
More space for group editing

### ISSUE THREAD

**Issue by justinkruit** [AdvancedCustomFields/acf#712]
The new layout is alright, but the "Reducing the Vertical Height of the Field Settings" fix has the opposite effect with the new sticky headers. On a 1920x1080 screen it's not even possible sometimes to see all the fields nicely. Please consider removing the sticky from `.acf-admin-toolbar` and `.acf-headerbar`. Even if only the sticky is being removed from `.acf-admin-toolbar` (since these are just shortcuts and not needed when editing the group) it woud create more needed space.

![image](https://user-images.githubusercontent.com/1642830/191540934-f1835db3-0565-4efc-abb4-ff17e6ac0f02.png)
![image](https://user-images.githubusercontent.com/1642830/191541255-a64219bc-675e-4050-aad0-3bb329239ef5.png)

**Comment by justinkruit** [comment]
And for people that want a quick fix for this, I included this css to my admin interface to make it a bit cleaner.
```
#wpcontent .acf-admin-toolbar {
  position: unset;
}
.acf-headerbar-field-editor {
  top: 32px;
  min-height: 0;
}
```

**Comment by polevaultweb** [comment]
Thanks @justinkruit - we'll get this improved in the next release.

**Comment by Humanify-nl** [comment]
I have to agree that having a double top bar of 144 pixels that takes almost 7,5% off the screen real estate is indeed a bit overkill.

How about consolidating both into one? It is a whole bar of 72px for a 'add field' button.

**Comment by black-eye** [comment]
+1
